sentinel-extension/sentinel-logging-extension-slf4j/src/main/java/com/alibaba/csp/sentinel/logging/slf4j/CommandCenterLogLogger.java

```diff
@@ -0,0 +1,66 @@
+package com.alibaba.csp.sentinel.logging.slf4j;
```

Comment:
Could you please add the license header?

Reply:
added